central/graphql/resolvers/images.go

```diff
@@ -153,6 +153,7 @@ func (resolver *imageResolver) TopImageVulnerability(ctx context.Context, args R
 // ImageVulnerabilities returns, as ImageVulnerabilityResolver, the vulnerabilities for the image
 func (resolver *imageResolver) ImageVulnerabilities(ctx context.Context, args PaginatedQuery) ([]ImageVulnerabilityResolver, error) {
 	defer metrics.SetGraphQLOperationDurationTime(time.Now(), pkgMetrics.Images, "ImageVulnerabilities")
+	// TODO(ROX-28320): Data here needs to be grouped by CVE not the ID as is done in the Image Vulnerabilities resolver.
```

Comment:
+1 . Without that we would likely see a lot of duplicated CVEs under image detail and deployment detail pages

Reply:
We do in fact see that when using graphql against the flattened model.  Working through that now on another PR.

Reply:
It is actually looking like the best path will be yet another resolver.